example/lib/main.dart

```diff
@@ -223,6 +224,16 @@ class _HomePageState extends ConsumerState<HomePage> {
                         setState(() {});
                       },
                     ),
+                    FloatingActionButton(
+                      heroTag: 'swipe',
+                      child: Icon(
+                          swipeable ? Icons.swipe : Icons.touch_app_outlined),
+                      onPressed: () {
+                        // Programmatically toggle the Navbar visibility
```

Comment:
This comment looks irrelevant of what the code does so it can be removed or corrected